specification/_types/Knn.ts

```diff
@@ -38,6 +38,8 @@ export interface KnnQuery {
   boost?: float
   /** Filters for the kNN search query */
   filter?: QueryContainer | QueryContainer[]
+  /** The minimum similarity for a vector to be considered a match */
+  similarity?: double
```

Comment:
```suggestion
  similarity?: float
```

According to [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/8.10/search-search.html) it should be `float`.

Reply:
thank you, changed it